### PR TITLE
[Cisco Umbrella] Add Audit Log support

### LINKED ISSUE #2295: Cisco Umbrella Enhancements
Our current Umbrella integration is currently experimental, and requires some enhancements before we move towards beta/GA. Cisco managed buckets and V6 logs are now supported, and have been removed from the original blockers list. 

Remaining items:

- **Audit logs:** the integration currently supports Proxy, Cloud Firewall, IP Logs and DNS logs. The [Audit log](https://docs.umbrella.com/deployment-umbrella/docs/admin-audit-log-report) records changes made by admins within Umbrella and should be added to our integration. 

- **AWS Role Support** add AWS Role support for authentication within our [fileset settings](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-cisco.html#_umbrella_fileset_settings).

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

Add Audit Logs support

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Closes #2295

### CODE REVIEW

File: packages/cisco_umbrella/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -131,6 +131,23 @@ processors:
...
+  - csv:
+      field: event.original
+      target_fields:
+        - event.id
+        - cisco.umbrella._tmp.time
+        - user.email
+        - user.name
+        - cisco.umbrella.audit.type
+        - event.action
+        - source.address
```
Comment (r00tu53r) [inline packages/cisco_umbrella/data_stream/log/elasticsearch/ingest_pipeline/default.yml:146]: The [doc](https://docs.umbrella.com/deployment-umbrella/docs/log-formats-and-versioning#audit) says it contains User IP. Could it be converted and stored into `source.ip` instead ?

File: packages/cisco_umbrella/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.7.0"
+  changes:
+    - description: Add Audit Logs
+      type: bugfix
```
Comment (r00tu53r) [inline packages/cisco_umbrella/changelog.yml:5]: `bugfix` -> `enhancement` ?

**Comment by r00tu53r** [comment]
Sample auditlog from my sandbox environment
[auditlog.csv](https://github.com/elastic/integrations/files/8675254/auditlog.csv)
.

File: packages/cisco_umbrella/data_stream/log/elasticsearch/ingest_pipeline/default.yml (hunk 1 above)
Comment (legoguy1000) [reply packages/cisco_umbrella/data_stream/log/elasticsearch/ingest_pipeline/default.yml:146]: There is a convert processor below that copies source.address to source.ip.

File: packages/cisco_umbrella/changelog.yml (hunk 2 above)
Comment (legoguy1000) [reply packages/cisco_umbrella/changelog.yml:5]: Ya, copy and paste error, I'll fix.

**Review by r00tu53r — APPROVED** [review]

**Comment by legoguy1000** [comment]
@efd6 can this be merged?

**Pull request merged**